.claude/agents/portfolio-surveyor.md

```
@@ -513,7 +523,24 @@ public and private — no per-repo loop needed to enumerate):
+     compares unrelated dependencies: two *first* failures of two different dependencies would read
+     as one 2-run streak and escalate to `(REPEATED — ACTIONABLE)`, while an unrelated green would
+     break a streak that is genuinely consecutive for the dependency that is actually broken. The
+     `name` is the logical unit whose history means anything. Code scanning is unaffected — its
+     `workflow_id` carries a single stable name, so grouping by name is identical there and strictly
+     more correct here:
+
+     ```sh
+     gh api --paginate \
+       "repos/devantler-tech/<repo>/actions/workflows/<workflow_id>/runs?branch=main&per_page=100" \
+       --jq '[.workflow_runs[] | select(.name == "<name>")]'
+     ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)